git-hg/updateforesttags.sh

```diff
@@ -60,91 +61,91 @@ if ! which git-remote-hg 2>/dev/null; then
 fi
 
 # Clone current AdoptOpenJDK repo
-cd $WORKSPACE/$GITHUB_REPO
-echo "Clone current $GITHUB_REPO"
-git clone $GITHUB_PROJECT/$GITHUB_REPO.git
-cd $GITHUB_REPO
+cd "$WORKSPACE"/"$GITHUB_REPO" || exit
+echo Clone current "$GITHUB_REPO"
+git clone "$GITHUB_PROJECT"/"$GITHUB_REPO".git
+cd "$GITHUB_REPO" || exit
 git fetch --tags
 OLDTAG=$(git describe --abbrev=0 --tags) 
 echo "Current openjdk level is $OLDTAG"
 
 # Clone current openjdk from Mercurial
 echo "Get base openjdk repository" 
-cd $WORKSPACE/openjdk/mirror
+cd "$WORKSPACE"/openjdk/mirror || exit
 git init
 git clone --bare "hg::${HG_REPO}"
 
-cd "$OPENJDK_VERSION.git"
+cd "$OPENJDK_VERSION.git" || exit
```

Comment:
This is slightly different from the changes you've made earlier - compare "cd "$OPENJDK_VERSION".git` and `cd $OPENJDK_VERSION.git"` (although granted this is just one level)

I tend to prefer using quotes around the whole thing as it looks a but cleaner so perhaps use things like `cd "$WORKSPACE/openjdk/mirror"` in all cases?

Reply:
Agreed, changing...